src/sass/_storage.scss

```diff
@@ -22,6 +40,34 @@
   button {
     margin-top: -6px;
   }
+
+  .volume-name-link {
+    max-width: 90%;
+  }
+
+  .u-row:hover {
+    background-color: $colors--light-theme--background-hover;
+  }
+
+  .actions-list {
+    margin-top: -7px;
```

Comment:
I think this is not needed here, it looks better aligned without, no?

Reply:
this may have been added in when I was battling with the styling and then forgotten... it does look better without it